Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -22,13 +22,21 @@ def install
     (buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve").install Dir["{*,.git,.gitignore}"]
```

Comment:
`make lib9p` job will be run `git submodule update --init`.
In Homebrew, will occur
```bash
==> make lib9p
git submodule update --init
fatal: no submodule mapping found in .gitmodules for path 'vendor/lib9p'
mk/lib9p.mk:36: recipe for target 'vendor/lib9p/%.c' failed
make: *** [vendor/lib9p/%.c] Error 128
```

So, needs fix `Dir["{*,.git,.gitignore}"]` -> ` Dir["{*,.git,.gitignore,.gitmodules}"]`